### PR TITLE
docs: :pencil2: small edits to architecture and naming page

### PR DESCRIPTION
**Pull request by lwjohnst86**
## Description

This PR is tidying some text up in the landing pages of design and architecture.

This PR needs a quick review.

### CODE REVIEW

File: docs/design/architecture/index.qmd [hunk 1]
```diff
@@ -1,16 +1,14 @@
...
+This documentation contains the architectural designs that are specific
+to Sprout. For designs on Seedcase overall, see the
+[Design](https://design.seedcase-project.org) documentation.
 
-## Purpose
-
-The core purpose of writing this architecture documentation is to be
-able to describe to ourselves and others what Seedcase Sprout is aiming
-to accomplish and how it will do that.
+The core purpose of these architectural documents are to be able to
```
Comment (martonvago) [inline docs/design/architecture/index.qmd:9]:
```suggestion
The core purpose of these architectural documents is to be able to
```

File: docs/design/architecture/index.qmd [hunk 2]
```diff
@@ -1,16 +1,14 @@
 ---
-title: "Design of Sprout"
+title: "Architecture"
 ---
 
-This documentation contains the design specific to Sprout, for design
-of the Seedcase platform, see the [Design](https://design.seedcase-project.org)
-documentation.
+This documentation contains the architectural designs that are specific
+to Sprout. For designs on Seedcase overall, see the
```
Comment (martonvago) [inline docs/design/architecture/index.qmd:6]:
```suggestion
to Sprout. For design details on Seedcase as a whole, see the
```
Just trying to rewrite "designs on"

File: docs/design/index.qmd [hunk 3]
```diff
@@ -1,8 +1,26 @@
...
+The core aim of Sprout is to **take data and metadata and convert them
+into a standardized and organized storage structure** that follows best
+practices for data engineering, at least within a research context.
+Specifically, Sprout aims to:
+
+1.  Take generated data from various source locations (such as the
+    clinic or laboratories), that may be distributed geographically or
+    organizationally, and store it in a standardized and efficient
+    format.
+2.  Require metadata be included for the data and organized following a
```
Comment (martonvago) [inline docs/design/index.qmd:14]:
```suggestion
2.  Ensure that metadata is included for the data and organized following a
```

File: docs/design/index.qmd [hunk 4]
```diff
@@ -1,8 +1,26 @@
...
+    standardized format, in a way that explicitly and programmatically
+    links the metadata to the data.
+
+The purpose of these documents are to describe the overall design of
```
Comment (martonvago) [inline docs/design/index.qmd:18]:
```suggestion
The purpose of these documents is to describe the overall design of
```

File: docs/design/index.qmd [hunk 5]
```diff
@@ -1,8 +1,26 @@
...
+Sprout in enough detail to primarily help us develop Sprout in a way
```
Comment (martonvago) [inline docs/design/index.qmd:19]:
```suggestion
Sprout in enough detail to help us develop Sprout in a way
```

**Review by martonvago — CHANGES_REQUESTED** [review]
Nice!!! 👍👍

File: docs/design/index.qmd [hunk 6]
```diff
@@ -1,8 +1,26 @@
 ---
-title: "Design of Sprout"
+title: "Design"
 ---
 
-Within the design documentation are two main sections:
+The core aim of Sprout is to **take data and metadata and convert them
+into a standardized and organized storage structure** that follows best
+practices for data engineering, at least within a research context.
```
Comment (signekb) [inline docs/design/index.qmd:7]:
```suggestion
practices for data engineering, particularly within a research context.
```
or 
```suggestion
practices for data engineering, particularly with a focus on research contexts.
```

File: docs/design/index.qmd [hunk 7]
```diff
@@ -1,8 +1,26 @@
...
+Specifically, Sprout aims to:
+
+1.  Take generated data from various source locations (such as the
+    clinic or laboratories), that may be distributed geographically or
+    organizationally, and store it in a standardized and efficient
+    format.
```
Comment (signekb) [inline docs/design/index.qmd:13]:
```suggestion
1.  Take generated data from various source locations (such as
    clinics or laboratories), which may be distributed geographically or
    organizationally, and store it in a standardized and efficient
    format.
```

File: docs/design/index.qmd [hunk 8]
```diff
@@ -1,8 +1,26 @@
...
+2.  Ensure that metadata is included for the data and organized following a
+    standardized format, in a way that explicitly and programmatically
+    links the metadata to the data.
```
Comment (signekb) [inline docs/design/index.qmd:16]:
```suggestion
2.  Ensure that metadata is included for the data and organized in a
    standardized format, explicitly and programmatically
    linking the metadata to the data.
```

File: docs/design/index.qmd [hunk 9]
```diff
@@ -1,8 +1,26 @@
...
+
+The purpose of these documents is to describe the overall design of
+Sprout in enough detail to help us develop Sprout in a way
+that is maintainable over the long term and that ensures we as a team
+are on the same page on what Sprout is and is not.
```
Comment (signekb) [inline docs/design/index.qmd:21]:
```suggestion
The purpose of these documents is to describe the overall design of
Sprout in enough detail to help us develop it in a way
that is sustainable (i.e., maintainable over the long term) and that ensures we as a team
have a shared understanding of what Sprout is and is not.
```

**Review by signekb — CHANGES_REQUESTED** [review]
Very nice 👍  I have included a couple of suggestions.

**Pull request merged**